Formula/netclient.rb

```diff
@@ -0,0 +1,34 @@
+class Netclient < Formula
+  desc "Automate fast, secure, and distributed virtual networks using WireGuard"
+  homepage "https://netmaker.io"
+  url "https://github.com/gravitl/netmaker/archive/v0.15.0.tar.gz"
+  sha256 "cd6f595365dd409400a61b6e7220133d0ea281db1ecc38ac541bc929f748d53e"
+  license "SSPL-1.0"
+  head "https://github.com/gravitl/netmaker.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "wireguard-tools"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X 'main.version=v#{version}'"), "netclient/main.go"
+  end
+
+  def caveats
+    <<~EOS
+      Netclient requires a daemon to function properly.
+      This daemon needs to run as root, and can be enabled using:
+        sudo brew services start netclient
+    EOS
+  end
+
+  service do
+    run [opt_bin/"netclient", "daemon"]
+    keep_alive true
+    environment_variables PATH: std_service_path_env
+  end
+
+  test do
+    output = pipe_output "'#{bin}/netclient' 2>&1"
```

Comment:
Do we need to pass an empty string via stdin here? That's what `pipe_output` does (with no additional arguments, at least). If not:
```suggestion
    output = shell_output("#{bin}/netclient 2>&1")
```
Also, why does this exit with `0` when not run as root? That seems odd.

Reply:
The reason I used `pipe_output` with `result = nil` is exactly to swallow the error code `1`. If I use `shell_output`, it will by default assert exit code `0`, which fails the test.

Reply:
So why not
```suggestion
    output = shell_output("#{bin}/netclient 2>&1", 1)
```
?

Reply:
Oh, I didn't realize `shell_output` also supports the exit code parameter. Change accepted, thanks!